src/util/container-request.js

```diff
@@ -1,58 +1,85 @@
 import {detectContainerForBytes, getId3Offset} from '@videojs/vhs-utils/dist/containers';
-import {stringToBytes, toUint8} from '@videojs/vhs-utils/dist/byte-helpers';
+import {stringToBytes, concatTypedArrays} from '@videojs/vhs-utils/dist/byte-helpers';
 import {callbackWrapper} from '../xhr';
 
+// calls back if the request is readyState DONE
+// which will only happen if the request is complete.
+const callbackOnCompleted = (request, cb) => {
+  if (request.readyState === 4) {
+    return cb();
+  }
+  return;
+};
+
 const containerRequest = (uri, xhr, cb) => {
-  let bytes;
+  let bytes = [];
   let id3Offset;
+  let finished = false;
+
+  const callback = function(err, req, type, _bytes) {
```

Comment:
Minor, but I find it a bit hard to read when there are two params that are essentially the same name, but have different spellings (e.g., `cb` and `callback`). Since this one serves the purpose of finishing the request, maybe we can name it something along the lines of `endRequestAndCallback`.

Reply:
`requestCompletedCallback` here
and on line 14 `cb` to `containerRequestCallback`

Reply:
Done!